File: app/src/main/java/com/congda/tianjianxin/interfaces/KeyBoardHeightChangeListener.java
package com.congda.tianjianxin.interfaces;

/**
 * Created by cxf on 2018/11/8.
 */

public interface KeyBoardHeightChangeListener {
    void onKeyBoardHeightChanged(int visibleHeight, int keyboardHeight);

    boolean isSoftInputShowed();
}

File: app/src/main/java/com/congda/tianjianxin/interfaces/KeyBoardHeightChangeListener.kt
package com.congda.tianjianxin.interfaces

/**
 * Created by cxf on 2018/11/8.
 */
interface KeyBoardHeightChangeListener {
    fun onKeyBoardHeightChanged(visibleHeight: Int, keyboardHeight: Int)
    val isSoftInputShowed: Boolean
}